ompi/mca/common/monitoring/Makefile.am

```diff
@@ -13,11 +13,15 @@
 # $HEADER$
 #
 
+include $(top_srcdir)/Makefile.ompi-rules
```

Comment:
The commit message on this commit no longer matches what the commit actually does.

I don't know / care if you want to split this into multiple commits:
* Add the dependencies
* Fix the comments
* Use Makefile.ompi-rules